x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/StartDatafeedAction.java

```diff
@@ -190,6 +190,9 @@ public static DatafeedParams parseRequest(String datafeedId, XContentParser pars
 
         public DatafeedParams(String datafeedId, long startTime) {
             this.datafeedId = ExceptionsHelper.requireNonNull(datafeedId, DatafeedConfig.ID.getPreferredName());
+            if (startTime < 0) {
+                throw new IllegalArgumentException("[startTime] must not be negative [" + startTime + "].");
```

Comment:
In the rest API this field is referred to as `start` we tend to use the API names in these sort of validation methods and there is a member in the class that defines the name


```suggestion
                throw new IllegalArgumentException("[" + START_TIME.getPreferredName() + "] must not be negative [" + startTime + "].");
```


https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-start-datafeed.html